packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -688,6 +688,63 @@ class EditableText extends StatefulWidget {
   /// Called when the user indicates that they are done editing the text in the
   /// field.
   /// {@endtemplate}
+  ///
+  /// {@tool snippet --template=stateful_widget_material}
+  /// When a non-completion action is pressed, such as "next" or "previous", it
+  /// is often desirable to move the focus to the next or previous field.  To do
+  /// this, handle it as in this example, by calling [FocusNode.focusNext] in
```

Comment:
Shouldn't this be `.nextFocus` not `.focusNext` as per L713?

https://github.com/flutter/flutter/pull/35926/files#diff-2d74aabcc309c260810a6cca572e134eR713

Reply:
Yep, thanks for catching that!  I'll fix it in another PR.